### DISCUSSION TITLE
Allow seeding from YAML (as in source code) inside ./extensions/seed

### DISCUSSION THREAD

**Discussion by gremo** [directus/directus#4807] [Feature Requests]
Probably everyone wants to seed the database for "frequently used" collections, when creating a new project based on Directus. 

[Here](https://github.com/directus/directus/blob/main/api/src/database/seeds/run.ts#L35) is were the seed happens inside Directus. Why not allowing us to define seed files inside i.e. `./extensions/seeds`, just like migrations?

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
